index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="./style.css">
+	<title>Original Trombone</title>
+</head>
+<body>
+	<header>
+		<img src="./img/original_trombones.png" alt="Original Trombones">
+		<nav id="nav-bar">
+			<ul>
+				<li>
+					<a class="nav-link" href="#features">Features</a>
+				</li>
+				<li>
+					<a class="nav-link" href="#work-pattern">Work Pattern</a>
+				</li>
+				<li>
+					<a class="nav-link" href="#pricing">Pricing</a>
+				</li>
+			</ul>
+		</nav>
+	</header>
+
+
+		<section id="hero">
+			<h2>Handcrafted, home-made masterpieces</h2>
+			<form id="form" action="https://www.freecodecamp.com/email-submit">
+			<input type="email" name="email" id="email" class="form-control" placeholder="Enter your email" required><br>
+				<input type="submit" name="submit" value="Get Started" class="btn-primary" >
+			</form>
+		</section>
+
+
+
+		<section id="features">
+			<div class="rows-1">
+				<h2>Premium Materials</h2>
+				<p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</p>
+			 </div>
+
+			<div class="rows-2">
+				<h2>Fast Shipping</h2>
+				<p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</p>
+			</div>
+
+
+			<div class="rows-3">
+				<h2>Quality Assurance</h2>
+				<p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</p>
+			</div>
+
+			</section>
+
+
+		<section id="work-pattern">
+		<iframe width="560" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe
+		>
+		</section>
+
+
+		<section id="pricing">
+		<div class="product-1">
+			<h2>Tenor Trombone</h2>
+			<h2>$600</h2>
+			<ul>
+				<li>Very Fast</li>
+				<li>Very Efficient</li>
+				<li>Easy To Use</li>
+			</ul>
+		</div>
+
+		<div class="product-2">
+			<h2>Bass Trombone</h2>
+			<h2>$500</h2>
+			<ul>
+				<li>Very Fast</li>
+				<li>Very Efficient</li>
+				<li>Easy To Use</li>
+			</ul>
+		</div>
+
+		<div class="product-3">
+			<h2>Valve Trombone</h2>
+			<h2>$700</h2>
+			<ul>
+				<li>Very Fast</li>
+				<li>Very Efficient</li>
+				<li>Easy To Use</li>
+			</ul>
+		</div>
+
+		</section>
```

Comment:
What do you think about wrapping this entire section in the `<main>`? This will make the code look more structured 💯 and semantic with the `<header>`, `<main>`, and `<footer>` sections being present. You can read more on the `<main>` tag [here](https://www.w3schools.com/tags/tag_main.asp).

Reply:
It is always better to include an explanation, a "why", for each requested change. For example:

This is a well-explained request: **Why don't you use a `<main>` tag to wrap this entire section, that would improve the HTML structure and make the page more accessible. You can read more on the `<main>` tag [here](https://www.w3schools.com/tags/tag_main.asp)**

_This request lacks a "why": Use the `<main>` tag to wrap this entire section. You can read more on the `<main>` tag [here.](https://www.w3schools.com/tags/tag_main.asp)_